lib/cosmos/system/system.rb

```diff
@@ -60,7 +60,7 @@ class System
     instance_attr_reader :limits_set
 
     # Known COSMOS ports
-    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED']
+    KNOWN_PORTS = ['CTS_API', 'TLMVIEWER_API', 'CTS_PREIDENTIFIED', 'CTS_CMD_ROUTER']
```

Comment:
CTS_PREIDENTIFIED routes out telemetry but it also accepts commands as input.   I'm sure there is a better name, but calling it a TLM router is inaccurate.  Normal routers output telemetry and accept commands.   The new Cmd router only outputs commands, and does not accept anything.  Maybe I am wrong, but I think it is ok that the only way COSMOS accepts telemetry is from interfaces.  Now we have:

All telemetry out stream
All commands out stream
All commands in stream (provides an alternative to the API and supports chaining servers)
Commands to one or more targets from an interface
Telemetry from one or more targets from an interface

We don't have:
All telemetry in stream 

Might be useful for testing?  I'm not convinced it is a good thing and wouldn't be abused.